vite.config.ts

```diff
@@ -2,17 +2,28 @@ import { defineConfig } from 'vite';
 import react from '@vitejs/plugin-react';
 import visualizer from 'rollup-plugin-visualizer';
 import pkg from './package.json';
+import { Plugin } from 'vite';
 
 /** Get build directory from package.json */
 const outDir = pkg.module.split('/')[0];
 
 // https://vitejs.dev/config/
 export default defineConfig({
+	esbuild: {
+		// These values
```

Comment:
is there something missing from this  comment?

Reply:
```suggestion
		// When using the “classic” JSX runtime in the react plugin,
		// we need to set these options: https://esbuild.github.io/api/#jsx-dev
```